docs/deployment/config_descriptions.md

```diff
@@ -211,6 +212,44 @@ Configuration for the Amazon Elasticsearch Service (ES) instance.  You can updat
           comparison_operator: GreaterThanThreshold
           threshold: '{{es.instanceCount}}'
           metric: Nodes
+```
+
+## sns
+Cumulus supports configuration and deployment of SNS topics and subscribers using `app/config.yml`. In the following code snippets we'll see an example topic and subscriber configuration.
+
+```yaml
+sns:
+  # this topic receives all the updates from
+  # step functions
+  sftracker:
+    subscriptions:
+      lambda:
+        endpoint:
+          function: Fn::GetAtt
+          array:
+            - sns2elasticsearchLambdaFunction
+            - Arn
+        protocol: lambda
+```
+
+The above code is an example of configuration for an SNS topic that will be called `sftrackerSns` in the resulting `cloudformation.yml` file. More information for each of the individual attributes can be found in [AWS SNS Topic Documentation](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-sns-topic.html).
+
+```yaml
+# sns: ...
+  sftrackerSubscription:
+    arn:
+      Fn::GetAtt:
+        - sftrackerSns
+        - Arn
+      endpoint:
+        function: Fn::GetAtt
+          array:
+            - someOtherLambdaFunction
+            - Arn
+        protocol: lambda
+```
+
+This snippet is an example of configuration for a list of SNS Subscriptions. The main difference between this and the previous example is the inclusion of the `sns.arn` attribute - this tells our deployment/compiling step that we're configuring subscriptions, not a new topic. More information for each of the individual attributes can be found in [AWS SNS Subscription Documentation](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-sns-subscription.html).
```

Comment:
I think it would be helpful to mention a couple things here:

- We're adding a new subscription for an existing lambda function to an existing SNS topic
- The resource names for the SNS topic and someOtherLambdaFunction have to refer to resources that already exist for your stack

Reply:
https://github.com/nasa/cumulus/pull/943/commits/191a785964971a1312c0cae59d681abcecccc645